yarn.lock

```diff
@@ -5651,6 +5632,11 @@ rfc4648@^1.1.0, rfc4648@^1.3.0, rfc4648@^1.4.0:
   resolved "https://registry.yarnpkg.com/rfc4648/-/rfc4648-1.4.0.tgz#c75b2856ad2e2d588b6ddb985d556f1f7f2a2abd"
   integrity sha512-3qIzGhHlMHA6PoT6+cdPKZ+ZqtxkIvg8DZGKA5z6PQ33/uuhoJ+Ws/D/J9rXW6gXodgH8QYlz2UCl+sdUDmNIg==
 
+rfc4648@^1.5.0:
```

Comment:
We should run `yarn-deduplicate` in here, since I see some redundancy creeping in.